README.md

```diff
@@ -129,6 +129,9 @@ When this command has finished, some secrets will have been copied to your
 * The administrative `admin.conf` file of the cluster has been copied in
   root of the `company-cluster` folder
 
+Also note that the `kubeconfig` configuration required by `kubectl` and other
```

Comment:
seems to be a duplicate of line 129? maybe merge them?

Reply:
hmm, you may be right that I could rephrase and combine those...  I wanted to be sure it was clear.

Reply:
I think, good to note that admin.conf is same as kubeconfig file. So using kubectl, users can choose either --kubeconfig flag <pathTo>/company-cluster/admin.conf or set environment variable such as `export KUBECONFIG=<pathTo>/my-company/admin.conf`

Reply:
Yes, I agree that it is good to replace 129 with your comment.